### DISCUSSION TITLE
page.meta.hide is an iterable.... what is it used for?

### DISCUSSION THREAD

**Discussion by divSelector** [squidfunk/mkdocs-material#6971] [Q&A] [answered]
Topic.

**Comment by alexvoss** [comment, marked as answer]
Generally, `page.meta` contains the metadata in your page header. Specifically, the `hide` attribute allows you to [hide navigation on a per-page basis](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=hide#hiding-the-sidebars).

**Reply by divSelector** [reply]
makes sense. I was expecting it to be a bool.
